sanic/app.py

```diff
@@ -1256,7 +1256,7 @@ def _helper(
             logger.setLevel(logging.DEBUG)
 
         if (
-            self.config.LOGO is not None
```

Comment:
You could possibly do
```python
self.config.LOGO not in [None, False, ""]
```

This should cover the cases in #1432

Reply:
The caveat is setting `self.config.LOGO` to `True` will literally print `True` (or similar).
Perhaps this could be handled by changing line 1262 to
```python
logger.debug(self.config.LOGO if self.config.LOGO is True else Config.LOGO)
```